### ISSUE TITLE
Temp working directories

### ISSUE THREAD

**Issue by ApkRE** [REAndroid/APKEditor#87]
APKEditor creates temp random named temp directory (tmp_xxxxxxxx) on every use and my dev directory is full of these which I have to keep deleting
Can you add an option to choose the temp working directory or automatically delete the working directory after use?

Many thanks for a great tool

**Comment by REAndroid** [comment]
Did you find .apk files inside those tmp directory ?
Which OS are you using ?

**Comment by ApkRE** [comment]
I'm using Windows
No files in them, just empty directories. 1 created each time I "merge" regardless of the output directory

**Comment by REAndroid** [comment]
I don't have windows machine, confirm 1.3.7 solves this issue

**Comment by ApkRE** [comment]
Still the same with 1.3.7
While merging directory tmp_1836e20c is created and apks extracted there
apks deleted once done, directory remains
Running again creates tmp_1836e20c_1

**Comment by REAndroid** [comment]
I updated to log the reason why directory not being deleted. Build your own or download from [Artifacts](https://github.com/REAndroid/APKEditor/actions/runs/8650098605/artifacts/1405832359).
Please test and let us know your results

**Comment by ApkRE** [comment]
I downloaded from your Artifacts link and tried again and it's still the same... :(

**Comment by REAndroid** [comment]
I was hopping you come back with logs

**Comment by REAndroid** [comment]
Thank you, i assumed you censored `*PackageName*` and not real value.

```
01.479 I: [MERGE] files.length = 2tmp_565db641\*PackageName*.apk
tmp_565db641\config.armeabi_v7a.apk
```

Supper odd for me! this two files were deleted even twice, I have to debug this  myself on windows

**Comment by ApkRE** [comment]
Yes I censored the package name, no problem with the name, same problem with all files...
Thanks :)
